.github/workflows/test.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       matrix:
         include:
         - python-version: "3.8"
+          toxenv: "min"
```

Comment:
A minor thing, but in general it seems better to have both 3.8 and min envs, because latest dependencies have a potential to break 3.8.